c/linux.c

```diff
@@ -8,6 +8,7 @@
 
 #define DFHASHSIZE 101
 #define MOUNTS "/proc/mounts"
+static const char *os_type = "Windows";
```

Comment:
I'm sorry, why os_type "Windows" in c/linux.c ?

Reply:
Hmm.. Ah. I don't think I meant for these commits to go in this PR. 

Regardless I was having odd link problems as the header declares the extern and it's never filled (even though its unused). A better solution is not declare the extern at all in the header for linux.